src/controllers/mail.controller.ts

```diff
@@ -64,3 +65,58 @@ export const getEmailsFromGmailHandler = async (
 		});
 	}
 };
+
+/**
+ * This controller will send a email from users gmail account
+ * @param req express request
+ * @param res express response
+ *
+ * @author aayushchugh
+ */
+export const postSendGmailHandler = async (
+	req: Request<PostSendGmailSchema["params"], {}, PostSendGmailSchema["body"]>,
+	res: Response,
+) => {
+	const { email } = req.params;
+	const { to, subject, html } = req.body;
+	const user = res.locals.user as User;
+
+	try {
+		const connectedAccount = user.connected_services.find((service) => service.email === email);
```

Comment:
Considering we'll be extracting `connectedAccount` in many `mail.controllers` from the request, wouldn't it make sense to just make it as a middleware to prevent copy-pasting the same code?

Reply:
I will create a new middleware than

Reply:
done